src/main/modules/nunjucks/index.ts

```diff
@@ -2,6 +2,7 @@ import * as path from 'path';
 import * as express from 'express';
 import * as nunjucks from 'nunjucks';
 import { i18n } from 'i18next';
+import { Paths as AppPaths } from 'main/app/paths';
```

Comment:
Why the re-assign?

Reply:
we will import case paths too in future and i want to import that as CasePaths.